### ISSUE TITLE
There is a 8MB corefile in the image !

### ISSUE THREAD

**Issue by rmanus** [cpoppema/docker-flexget#13]
```
# ls -l
total 8528
-rw-r--r--   1 root root   11831 Aug 21 18:10 =36
drwxr-xr-x   2 root root    4096 Aug 21 18:08 bin
drwxr-xr-x   2 root root    4096 Apr 12  2016 boot
drwxr-xr-x   2 abc  abc     4096 Aug 23 12:32 config
**-rw-------   1 root root 8646656 Jun 29  2016 core**
drwxr-xr-x   5 root root     380 Aug 23 12:31 dev
drwxr-xr-x   2 root root    4096 Aug 23 10:22 downloads
drwxr-xr-x 120 root root    4096 Aug 23 12:31 etc
drwxr-xr-x   2 root root    4096 Apr 12  2016 home
drwxr-xr-x  15 root root    4096 Aug 21 18:08 lib
drwxr-xr-x   2 root root    4096 Aug 21 18:07 lib64
drwxr-xr-x   2 root root    4096 Jun 29  2016 media
drwxr-xr-x   2 root root    4096 Jun 29  2016 mnt
drwxr-xr-x   2 root root    4096 Jun 29  2016 opt
dr-xr-xr-x 266 root root       0 Aug 23 12:31 proc
drwx------   6 root root    4096 Aug 23 12:31 root
drwxr-xr-x   6 root root    4096 Aug 23 12:31 run
drwxr-xr-x   2 root root    4096 Aug 21 18:08 sbin
drwxr-xr-x   2 root root    4096 Jun 29  2016 srv
dr-xr-xr-x  13 root root       0 Aug 23 12:31 sys
drwxrwxrwt   2 root root    4096 Aug 23 12:31 tmp
drwxr-xr-x  26 root root    4096 Aug 23 12:31 usr
drwxr-xr-x  23 root root    4096 Aug 23 12:31 var

# file core
core: ELF 64-bit LSB core file x86-64, version 1 (SYSV), SVR4-style, from 'auplink /mnt/scratch/var-lib-docker/aufs/mnt/f70abbe99683746b8236d4a214a77b27c2'

```

**Comment by cpoppema** [comment]
I am as suprised as you are ! It seems this file already exists in the base image I am using:

```
 $     docker run --rm -t -i phusion/baseimage:0.9.19 /sbin/my_init -- bash -l
*** Running /etc/my_init.d/00_regen_ssh_host_keys.sh...
*** Running /etc/rc.local...
*** Booting runit daemon...
*** Runit started as PID 9
*** Running bash -l...
root@18ce6d57505e:/# ls -l
total 8508
drwxr-xr-x   2 root root    4096 Oct  6  2016 bin
drwxr-xr-x   2 root root    4096 Apr 12  2016 boot
-rw------- 547 root root 8646656 Jun 29  2016 core
drwxr-xr-x   5 root root     400 Aug 23 13:31 dev
drwxr-xr-x   1 root root    4096 Aug 23 13:31 etc
drwxr-xr-x   2 root root    4096 Apr 12  2016 home
drwxr-xr-x   8 root root    4096 Oct  6  2016 lib
drwxr-xr-x   2 root root    4096 Oct  6  2016 lib64
drwxr-xr-x   2 root root    4096 Jun 29  2016 media
drwxr-xr-x   2 root root    4096 Jun 29  2016 mnt
drwxr-xr-x   2 root root    4096 Jun 29  2016 opt
dr-xr-xr-x 324 root root       0 Aug 23 13:31 proc
drwx------   4 root root    4096 Oct  6  2016 root
drwxr-xr-x   1 root root    4096 Aug 23 13:31 run
drwxr-xr-x   2 root root    4096 Oct  6  2016 sbin
drwxr-xr-x   2 root root    4096 Jun 29  2016 srv
dr-xr-xr-x  13 root root       0 Aug 23 13:31 sys
drwxrwxrwt   2 root root    4096 Jul  8  2016 tmp
drwxr-xr-x  10 root root    4096 Oct  6  2016 usr
drwxr-xr-x   1 root root    4096 Oct  6  2016 var
```

It looks like it is gone in next builds*. I will browse through the next few releases to see if I can upgrade the base image without much pain.

*From the next build after this one, it is no longer there:

```
 $     docker run --rm -t -i phusion/baseimage:0.9.20 /sbin/my_init -- bash -l
Unable to find image 'phusion/baseimage:0.9.20' locally
0.9.20: Pulling from phusion/baseimage
d54efb8db41d: Pull complete 
f8b845f45a87: Pull complete 
e8db7bf7c39f: Pull complete 
9654c40e9079: Pull complete 
6d9ef359eaaa: Pull complete 
a3ed95caeb02: Pull complete 
39a6e97e0cc9: Pull complete 
781d059f5a4e: Pull complete 
Digest: sha256:34ff7b7338e92856477cbe4c6f176bd2f1d3e2f85e19a38ea42beb3f30112935
Status: Downloaded newer image for phusion/baseimage:0.9.20
*** Running /etc/my_init.d/00_regen_ssh_host_keys.sh...
*** Running /etc/rc.local...
*** Booting runit daemon...
*** Runit started as PID 9
*** Running bash -l...
root@f3a0bc9cf71b:/# ls -l
total 68
drwxr-xr-x   4 root root 4096 Aug 23 13:32 bd_build
drwxr-xr-x   2 root root 4096 Aug 23 13:32 bin
drwxr-xr-x   2 root root 4096 Apr 12  2016 boot
drwxr-xr-x   5 root root  400 Aug 23 13:32 dev
drwxr-xr-x   1 root root 4096 Aug 23 13:32 etc
drwxr-xr-x   2 root root 4096 Apr 12  2016 home
drwxr-xr-x   8 root root 4096 Aug 23 13:32 lib
drwxr-xr-x   2 root root 4096 Aug 23 13:32 lib64
drwxr-xr-x   2 root root 4096 Feb 14  2017 media
drwxr-xr-x   2 root root 4096 Feb 14  2017 mnt
drwxr-xr-x   2 root root 4096 Feb 14  2017 opt
dr-xr-xr-x 325 root root    0 Aug 23 13:32 proc
drwx------   4 root root 4096 Aug 23 13:32 root
drwxr-xr-x   1 root root 4096 Aug 23 13:32 run
drwxr-xr-x   2 root root 4096 Aug 23 13:32 sbin
drwxr-xr-x   2 root root 4096 Feb 14  2017 srv
dr-xr-xr-x  13 root root    0 Aug 23 13:31 sys
drwxrwxrwt   2 root root 4096 Mar 21 10:33 tmp
drwxr-xr-x  10 root root 4096 Aug 23 13:32 usr
drwxr-xr-x   1 root root 4096 Aug 23 13:32 var
```

**Comment by cpoppema** [comment]
I have updated the base image to version 0.9.22. This has been fixed since 0.9.20 as can also be confirmed in issue 332 in the [phusion/baseimage-docker repo](https://github.com/phusion/baseimage-docker/issues/332). If you run `docker pull cpoppema/docker-flexget` and recreate your container it should no longer be there :).
